src/www/ui_tests/api/Models/AgentTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/*
+ SPDX-FileCopyrightText: © 2024 Valens Niyonsenga <valensniyonsenga2003@gmail.com>
+
+ SPDX-License-Identifier: GPL-2.0-only
+*/
+/**
+ * @file
+ * @brief Tests for FileInfo
+ */
+namespace Fossology\UI\Api\Test\Models;
+
+use Fossology\UI\Api\Models\Agent;
+use Fossology\UI\Api\Models\ApiVersion;
+
+class AgentTest extends \PHPUnit\Framework\TestCase
+{
+
+  /**
+   * Provides test data and an instance of the Agent class.
+   * @return array An associative array containing test data and an Agent object.
+   */
+  public function getAgentInfo($version=ApiVersion::V2)
+  {
+    if($version==ApiVersion::V1){
+      $agentInfo = [
+        'successfulagents' => [],
+        'uploadid' => 4,
+        'agentname' => "decisionimporter",
+        'currentagentid' => 24,
+        'currentagentrev' => "68097a",
+        'isagentrunning' => null
+      ];
+    }else{
+      $agentInfo = [
+        'successfulAgents' => [],
+        'uploadId' => 4,
+        'agentName' => "decisionimporter",
+        'currentAgentId' => 24,
+        'currentAgentRev' => "68097a",
+        'isAgentRunning' => null
+      ];
+    }
+    return [
+      'agentInfo' =>  $agentInfo,
+      'obj' => new Agent([], 4, "decisionimporter", 24, "68097a", null)
```

Comment:
Ditto, tests will need to be adjusted due to a change in the agent model as described above. (No valid key in V1 `return`).